dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/ui/UISoftwareResource.java

```diff
@@ -29,8 +29,9 @@
 import javax.ws.rs.core.Response;
 
 import org.icgc.dcc.portal.server.resource.Resource;
-import org.icgc.dcc.portal.server.service.StorageClientService;
-import org.icgc.dcc.portal.server.service.StorageClientService.MavenArtifactVersion;
+import org.icgc.dcc.portal.service.SoftwareService;
```

Comment:
Should be `org.icgc.dcc.portal.server.service` not `org.icgc.dcc.portal.service`